docs/tutorials/getting_started/install_templates/other_releases.md

```diff
@@ -0,0 +1,13 @@
+* To get a previous `IDAES release <https://github.com/IDAES/idaes-pse/releases>`_, for example v1.13 ::
+
+   pip install idaes-pse==1.13
+
+* To get the latest version from the GitHub main branch ::
+
+   pip install 'idaes-pse[prerelease] @ https://github.com/IDAES/idaes-pse/archive/main.zip'
+
+* To get a specific fork or branch, for example myfork (of idaes-pse) and mybranch ::
+
+   pip install 'idaes-pse[prerelease] @ https://github.com/myfork/idaes-pse/archive/mybranch.zip'
+
+* **For developers**: follow the :ref:`advanced user installation<tutorials/advanced_install/index:Advanced User Installation>`.
```

Comment:
Is this file still used?  I couldn't see it referenced in any of the other files.

Reply:
Oops. Artifact from a previous version. Removed!